app/enterprise/1.3-x/brain-immunity/troubleshooting.md

```diff
@@ -51,16 +55,16 @@ When the Collector App cannot communicate with Kong, the **kong_status** object
 ### Collector App is not able to communicate with Kong
```

Comment:
Is this the error text? If not, then maybe rephrase to "Collector App can't communicate with Kong"